README.md

```diff
@@ -20,15 +22,19 @@ Or, to install the latest package from PyPi.org:
 
     $ pip install audio-offset-finder
 
+You will need to [install FFmpeg](https://ffmpeg.org/download.html) to use the command-line tool, or to use the file-related functions in the library.
+
 Usage
 -----
 
+To use the command-line tool:
+
     $ audio-offset-finder --help
-    $ audio-offset-finder --find-offset-of file1.wav --within file2.wav
+    $ audio-offset-finder --find-offset-of file1.wav --within file2.wav --show-plot
```

Comment:
I'd recommend describing the plot options as a standalone set of examples:

> You can also produce a plot that shows the cross-correlatation measured:
>   $ audio-offset-finder --find-offset-of file1.wav --within file2.wav --show-plot
> or to save the plot to an image file:
>   $ audio-offset-finder --find-offset-of file1.wav --within file2.wav --save-plot plot_file.png

Reply:
Addressed in https://github.com/bbc/audio-offset-finder/pull/29/commits/cc401bfe5e2e5987f82118295281f7029348323c